src/lib/list/list.scss

```diff
@@ -262,6 +262,22 @@ $mat-list-item-inset-divider-offset: 72px;
   }
 }
 
+.mat-button-list {
+  //remove the native button look and make is look like a list item
+  button {
```

Comment:
Rather than using two levels of selector (`.mat-button-list button`), it would be slightly better to use a single css class (`.mat-action-list-item`) for this style. This will have a lower specificity and make it easier for users to customize the css.

Reply:
We discussed and decided to go with the current nav list pattern.